javascript/javascript1/week3/whenWillWeBeThere.js

```diff
@@ -0,0 +1,11 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+
+const timeInHours = travelInformation.destinationDistance / travelInformation.speed;
```

Comment:
Excellent variable naming! ⭐ The tasks did technically ask for a function to be implemented; I suspect that will be no issue for you to fix. 😄